libraries/AP_HAL_ChibiOS/hwdef/CarbonixPLBL496/hwdef-bl.dat

```diff
@@ -0,0 +1,63 @@
+# hw definition file for processing by chibios_pins.py
+
+# MCU class and specific type
+MCU STM32L496 STM32L496xx
+
+FLASH_RESERVE_START_KB 0
+FLASH_BOOTLOADER_LOAD_KB 36
+
+# reserve some space for params
+APP_START_OFFSET_KB 4
+
+# board ID for firmware load
+APJ_BOARD_ID 1066
+
+# setup build for a peripheral firmware
+env AP_PERIPH 1
+
+# crystal frequency; 0 - Internal clock
+OSCILLATOR_HZ 0
```

Comment:
did you really mean no crystal? is this temporary or planned for production use? I'm not certain CAN will be completely reliable with no crystal

Reply:
> please split into 3 commits, try using Tools/gittools/git-subsystems-split then re-work the commit msgs

Ok, will make it 3 commits.

Reply:
> did you really mean no crystal? is this temporary or planned for production use? I'm not certain CAN will be completely reliable with no crystal

I think Al decided to go ahead with no crystal. 
But completely agree with using crystal on production boards as it will definitely affect CAN with temp and voltage drift.